scripts/compose.py

```diff
@@ -1673,6 +1712,12 @@ def add_arguments(cls, parser):
             dest=cls.option_name() + '_agent_local_repo',
             help=cls.name() + " local repo path for agent"
         )
+        parser.add_argument(
```

Comment:
I could not use `--opbeans-elasticsearch-url` as a parameter because it is defined on every child class and causes a conflict in names, however, all parameter write the same option so you only have to set it once.

Reply:
This will probably be a pain.  A workaround is to remember if the option was set and skip it after the first time, like done here: https://github.com/elastic/apm-integration-testing/blob/0d4b122b2b1aaede49a6282083c7e4efb745e346/scripts/compose.py#L1359 (can't just `return` in this case but can guard with an `if`)

Reply:
finally, I make it global on the `start` parser definition makes more sense.